app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -1,21 +1,33 @@
 class OmniauthCallbacksController < ActionController::Base
+  REQUIRED_PARAMS = ['email', 'first_name', 'last_name'].freeze
+  METADATA_IGNORED_PARAMS = REQUIRED_PARAMS + ['fingerprint'].freeze
```

Comment:
I _think_ this is just freezing the `fingerprint` array, and not the resulting `METADATA_IGNORED_PARAMS` params. Might want to wrap in parens, or use `foo.concat(bar).freeze`

Reply:
You know, I thought about that and then thought, "eh it's concatenating two frozen arrays" but I suppose it's still mutable so I should fix it.